wsmaster/che-core-api-auth-bitbucket/src/main/java/org/eclipse/che/security/oauth1/NoopOAuthAuthenticator.java

```diff
@@ -49,7 +49,6 @@ public String computeAuthorizationHeader(String userId, String requestMethod, St
 
   @Override
   public String getLocalAuthenticateUrl() {
-    throw new RuntimeException(
-        "The fallback noop authenticator cannot be used for authentication. Make sure OAuth is properly configured.");
+    return "Empty URL";
```

Comment:
Why not to keep throwing RuntimeException ?

Reply:
To avoid the exception when the `NoopOAuthAuthenticator` is used to create an API client with an empty authenticator: https://github.com/eclipse-che/che-server/blob/17c82bb6f7d553e43d4ff2bedca40adb6e0cadd9/wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/BitbucketServerPersonalAccessTokenFetcher.java#L114